src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -66,6 +67,8 @@ const AnonymousChannelListScreen = ({route}) => {
         <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
 
+      <IncognitoEmptyChat totalChannel={anonChannels.length} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `IncognitoEmptyChat` component is used here with the `totalChannel` prop set to the length of `anonChannels`. Make sure that `anonChannels` is always an array to prevent runtime errors when accessing its length. If there's a chance it could be `null` or `undefined`, use a fallback value.
```diff
+ <IncognitoEmptyChat totalChannel={anonChannels?.length || 0} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->